Changes

```diff
@@ -19,6 +19,7 @@ Verilator 5.009 devel
 * Support $fopen as an expression.
 * Support class extends of package::class.
 * Change ZERODLY to a warning.
+* Fix false LATCH warning on --assert 'unique else if' (#4033). [Jesse Taube]
```

Comment:
This is fine, just note for next time you don't have to update this file, I can do it for you when I release, as otherwise it can cause merge conflicts.